java/streaming/src/main/java/com/ray/streaming/operator/impl/MasterOperator.java

```diff
@@ -0,0 +1,17 @@
+package com.ray.streaming.operator.impl;
+
+import com.ray.streaming.operator.OperatorType;
+import com.ray.streaming.operator.StreamOperator;
+
+
+public class MasterOperator extends StreamOperator {
```

Comment:
What is a MasterOperator?

Reply:
Master Operator is Controller Operator for dataflow

Reply:
Can you please add some documentation here?